analyses/copy_number_consensus_call/Snakefile

```diff
@@ -83,10 +83,10 @@ rule manta_filter:
         ## The first awk looks at column 6 to filter out for loss/gain (DEL/DUP). Then it prints out 6 of the 7 columns above. Put NA for both p-value and copy number since MANTA results don't have these values.
         ## the first awk also filters out for CNV length
         ## The last pipe is to sort first digit of chromosome number numerically
-        """awk '$6~/DEL/ {{if ($5 > {params.SIZE_CUTOFF}) {{print "chr"$2,$3,$4,$5,"NA","NA",$6}}}}' {input} """
+        """awk '$6~/DEL/ {{if ($5 > {params.SIZE_CUTOFF}) {{print "chr"$2,$3,$4,$5,"NA","NA","NA",$6}}}}' {input} """
```

Comment:
Is this the correct order? I would assume the added `NA` for manta should be at the end, as with freec?

Reply:
Never mind... I see my error now.

Reply:
It is the correct order. I had the same question, :) . For Cnvkit and Freec, I need to add in `"DUP"` or `"DEL"` as the last column using the 2nd `awk` command, so you see the `"NA"` last. However, for Manta, the 6th column of the raw file is the DUP or DEL, so it should go like `"NA","NA","NA",$6`.